common/common-api/src/main/java/org/ow2/proactive/utils/ObjectByteConverter.java

```diff
@@ -70,6 +75,9 @@
     public static final byte[] objectToByteArray(Object obj, boolean compress) throws IOException {
         ByteArrayOutputStream baos = null;
         ObjectOutputStream oos = null;
+        if (obj == null) {
```

Comment:
Shouldn't we let the method throws a NullPointerException in this case? and also all the similar null checks in this pull request?